ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -164,6 +164,8 @@ presubmits:
     cluster: build01
     context: ci/prow/azure-e2e-upgrade
     decorate: true
+    decoration_config:
+      timeout: 5h0m0s
```

Comment:
should the EUS upgrade job also get the bump?

https://github.com/openshift/release/pull/57325/files#diff-4eb058a875dca28e245af8a07a35af60cc794b29baafc5b3f2318e4d95468262R5